readme.md

```diff
@@ -167,7 +167,7 @@
 - [Fortran](https://github.com/rabbiabram/awesome-fortran#readme)
 - [PHP](https://github.com/ziadoz/awesome-php#readme)
 	- [Composer](https://github.com/jakoch/awesome-composer#readme) - Package manager.
-- [Delphi](https://github.com/Fr0sT-Brutal/awesome-delphi#readme)
+- [Delphi/FreePascal/(any)Pascal](https://github.com/Fr0sT-Brutal/awesome-pascal#readme)
```

Comment:
This should be the list name. Any clarification on what it covers should be in the description.

Reply:
I've no idea how to change this PR so I made a new one #1527